app/navigation/nonsipp/SharesDisposalNavigator.scala

```diff
@@ -25,6 +25,10 @@ import play.api.mvc.Call
 object SharesDisposalNavigator extends JourneyNavigator {
 
   override def normalRoutes: UserAnswers => PartialFunction[Page, Call] = userAnswers => {
+
+    case CompanyBuyerNamePage(srn, index, secondaryIndex) =>
+      controllers.routes.UnauthorisedController.onPageLoad()
+
```

Comment:
There is also a TODO further down to do the navigation from the `WhoWereTheSharesSoldToPage` which can be now uncommented (and anticipated controller name changed to `CompanyNameOfSharesBuyerController`